Source/WebCore/rendering/RenderBox.h

```diff
@@ -165,11 +165,18 @@ class RenderBox : public RenderBoxModelObject {
     void addLayoutOverflow(const LayoutRect&);
     void addVisualOverflow(const LayoutRect&);
     void clearOverflow();
+    RenderOverflow& ensureOverflow();
 
     void addVisualEffectOverflow();
     LayoutRect applyVisualEffectOverflow(const LayoutRect&) const;
 
-    void addOverflowFromInFlowChildOrAbsolutePositionedDescendant(const RenderBox&);
+
+    enum class ComputeOverflowOptions {
+        None,
+        RecomputeFloats = 1 << 0,
+    };
+    virtual void addOverflowFromInFlowChildren(OptionSet<ComputeOverflowOptions> = { });
+    void addOverflowFromChild(const RenderBox& child, OptionSet<ComputeOverflowOptions> = { });
```

Comment:
Alan points out these are not always children, so need to rename this to something else that's more correct.

Reply:
Renamed to addOverflowFromContainedBox